cli/command/target/add.go

```diff
@@ -84,10 +87,10 @@ func NewAddCommand(curveadm *cli.CurveAdm) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.StringVar(&options.host, "host", "localhost", "Specify target host")
-	flags.BoolVar(&options.create, "create", false, "Create volume iff not exist")
+	flags.BoolVar(&options.create, "create", false, "Create volume if not exist")
 	flags.StringVar(&options.size, "size", "10GB", "Specify volume size")
 	flags.StringVarP(&options.filename, "conf", "c", "client.yaml", "Specify client configuration file")
-
+	flags.StringVar(&options.blocksize, "blocksize", "512byte", "Specify volume block size")
```

Comment:
The default unit of blocksize should be `Byte`, and if we want to provide more unit for convenience, we can use `KB`、`MB` and etc, like:

```
--blocksize 512  # Byte, also default unit
--blocksize 1KB  # KB
--blocksize 1MB  # MB
```